node/risk-app/README.md

```diff
@@ -15,13 +15,51 @@
-|----------|-----------------------|:---------:|-------------|:---------------------------:|-------|
-| NODE_ENV | Node environment      |    no     | development | development,test,production |       |
+| name                        |      description                        | required  |   default   |            valid            | notes |
+|-----------------------------|-----------------------------------------|:---------:|-------------|:---------------------------:|-------|
+| NODE_ENV                    | Node environment                        |    no     | development | development,test,production |       |
+| RISK_APP_HOST               | Host server IP                          |   yes     |             |                             |       |
+| PORT                        | Server port #                           |   yes     |             |                             |       |
+| GEOSERVER_URL               | Geoserver API                           |   yes     |             |                             |       |
+| SERVICE_URL                 | Service API                             |   yes     |             |                             |       |
+| SIMULATE_ADDRESS_SERVICE    | Mocks address service for testing       |    no     | false       |                             |       |
+| HTTP_TIMEOUT_MS             | Timeout in ms                           |   yes     |             |                             |       |
+| G4_ANALYTICS_ACCOUNT        | GA ID                                   |    no     |             |                             |       |
+| GTAG_MANAGER_ID             | GTM ID                                  |    no     |             |                             |       |
+| FLOOD_WARNINGS_URL          | Flood warning API                       |   yes     |             |                             |       |
+| FLOOD_RISK_URL              | Flood risk API                          |   yes     |             |                             |       |
+| OS_POSTCODE_URL             | OS postcode API                         |   yes     |             |                             |       |
+| OS_CAPABILITIES_URL         | OS capabilites API                      |   yes     |             |                             |       |
+| OS_MAPS_URL                 | OS map API                              |   yes     |             |                             |       |
+| OS_NAMES_URL                | OS names API                            |   yes     |             |                             |       |
+| OS_SEARCH_KEY               | OS search key                           |   yes     |             |                             |       |
+| OS_MAPS_KEY                 | OS map key                              |   yes     |             |                             |       |
+| HTTP_PROXY                  |                                         |    no     |             |                             |       |
```

Comment:
This environment variable is in lower case `http_proxy` and the description is "Proxy that is used for external calls by the service"